patches/minecraft/net/minecraft/world/WorldProvider.java.patch

```diff
@@ -118,19 +119,34 @@
 +        return 1.0;
 +    }
 +
++    @Deprecated
 +    @Nullable
 +    @SideOnly(Side.CLIENT)
 +    public net.minecraftforge.client.IRenderHandler getSkyRenderer()
 +    {
 +        return this.skyRenderer;
 +    }
 +
++    @Deprecated
 +    @SideOnly(Side.CLIENT)
 +    public void setSkyRenderer(net.minecraftforge.client.IRenderHandler skyRenderer)
 +    {
 +        this.skyRenderer = skyRenderer;
 +    }
 +
++    @SideOnly(Side.CLIENT)
++    public net.minecraftforge.client.SkyRenderer getSkyRenderHandler()
++    {
++        return this.skyRenderHandler;
++    }
++
++    /** Sets sky renderer for certain pass and resource location. Renderer with higher priority is called earlier. */
++    @SideOnly(Side.CLIENT)
++    public void setSkyRenderer(net.minecraftforge.client.EnumSkyPass pass, net.minecraft.util.ResourceLocation id, @Nullable net.minecraftforge.client.IRenderHandler renderer, double priority)
```

Comment:
Should it rather be forced that `setSkyRenderer` should be called on the `SkyRenderer`? The method of this name implies that it's setting a field, and things will be overridden if you do it twice. (even with a different pass)
Or maybe a different name would be suitable? Maybe `setSkyPassRenderer` or `setSkyRendererPass`?

Reply:
I see, that'd be definitely better.